actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Added `deep_transform_keys` and `deep_transform_keys!` methods to StrongParameters.
```

Comment:
StrongParameters looks like a module name here. Could you either change it to "strong parameters", or the actual class name (`ActionController::Parameters`)?